content/en/docs/zero-code/java/spring-boot-starter/out-of-the-box-instrumentation.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Out of the box instrumentation
 description: Out of the box instrumentation for the starter
-aliases: [../annotations]
+aliases: [../instrumentation]
```

Comment:
do we want the alias to stay under `/spring-boot-starter/*`, e.g.
```suggestion
aliases: [instrumentation]
```

Reply:
That alias shouldn't be there in the first place. It's been dropped in #4563.